helmcharts/kitchen/install.sh

```diff
@@ -49,7 +49,7 @@ migrations)
 monitoring)
     rm -rf monitoring
     cp -rf ../obsrv monitoring
-    cp -rf ../services/{promtail,loki,grafana,kube-prometheus-stack,kafka-message-exporter,alert-rules} monitoring/charts/
+    cp -rf ../services/{promtail,loki,kube-prometheus-stack,kafka-message-exporter,alert-rules} monitoring/charts/
```

Comment:
why is grafana removed?

Reply:
its part of  kube-promethous-stack helm chart